optimum/graphcore/ipu_configuration.py

```diff
@@ -419,20 +509,51 @@ def validate_ipu_config(self):
                 matmul_proportion_mode_str = self._get_managed_attr_mode_name("matmul_proportion")
                 raise IncompatibleIPUConfigError(
                     f"{matmul_proportion_mode_str}={self._matmul_proportion} should use the"
-                    f" same number of IPUs as {ipus_per_replica_mode_str}={self._ipus_per_replica}"
+                    f" same number of IPUs as {ipus_per_replica_mode_str}={self._ipus_per_replica}."
                 )
 
-            # layers_per_ipu must have the same length as ipus per replica.
+            # Layers_per_ipu must have the same length as ipus per replica.
```

Comment:
This is a variable name, it shouldn't be capitalised
```suggestion
            # layers_per_ipu must have the same length as ipus per replica.
```